phpunit.xml.dist

```diff
@@ -3,8 +3,9 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
     bootstrap="./test/bootstrap.php"
-    colors="true">
-    <coverage processUncoveredFiles="true">
+    colors="true"
+    cacheDirectory=".phpunit.cache">
```

Comment:
Currently `.phpunit.result.cache` is listed in `.gitignore` - if this is a change with PHPUnit 10, can you update .gitignore instead, or make both things match?